test_schemas/en/test_list_collector_primary_and_two_list_collectors.json

```diff
@@ -0,0 +1,518 @@
+{
```

Comment:
Primary person is not relevant to this scenario. The scenario we need to create is, when you have a list collector and the list collector is not on the path. To do this, you do not need a primary person. I think the only thing you need is a driving question and a list collector.

Reply:
Isn't the scenario different to this though? It should be:

1) One list
2) Two list collectors, each with their own driving questions
3) Nothing added to list on 1st list collector
4) Item added to list on 2nd list collector
5) Change/remove links work for that list item

Reply:
Umm not sure, is it not simply ensuring we generate a link based on the list collector that is available or driving question if list collector is not available. Having 2 list collectorws or 3 won't matter, it is the fact that, the available one is chosen?. Also, we don't have requirements for 2 driving questions and that prob opens up a bunch of other scenarios we haven't thought about.

Reply:
You are correct re the need for only one ListColectorDrivingQuestion, but is there not still a need for two ListCollectors, as the issue that we have identified is that the add/remove links are being generated for the first list collector, which isn't on the path

Reply:
As discussed @JamesGardiner, the change and remove links are only relevant when there is at least 1 item on the list, which is only possible by having more than 1 list collector, therefore the driving question is not required. 

A list collector with a primary person collector is already handled I believe, so to test this scenario for change/remove links, I believe what you need is just 2 list collectors as James mentioned. (Primary person collector is not needed).


@petechd

Reply:
Ok, two list collectors now, thanks!